scripts/appleseedMaya/renderGlobals.py

```diff
@@ -518,7 +518,7 @@ def create(self):
                                 minValue=0.0001,
                                 fieldMinValue=0.0,
                                 maxValue=2.0,
-                                fieldMaxValue=10000.0,
+                                fieldMaxValue=25.0,
```

Comment:
Same question here.

Reply:
It is just limited to 25 in the UI, which is very high already. It can be overridden with MEL or Python if one would need to up to 10000, which is the hard limit set in the renderGlobals.cpp

Reply:
Ok.